src/main/webapp/app/config/constants/constants.ts

```diff
@@ -424,3 +424,5 @@ export const DEFAULT_TOAST_ERROR_OPTIONS: ToastOptions = {
 export const KEYCLOAK_LOGOUT_REDIRECT_PARAM = 'post_logout_redirect_uri';
 export const KEYCLOAK_SESSION_TERMINATED_PARAM = 'session_terminated';
 export const KEYCLOAK_UNAUTHORIZED_PARAM = 'unauthorized';
+
+export const REFERENCE_IDENTIFIERS = ['PMID: ', 'Abstract: ', 'NCT'];
```

Comment:
We used to can support `PMID` `PMID:` `PMID ` because of the flexibility of the regex. Now we have to have `PMID: ` in the text. I don't think this is ok.

Reply:
That's good to know. Always thought it had one type of format since most are written in that way. It can be fixed with a regex.